lib/topological_inventory/orchestrator/worker.rb

```diff
@@ -104,15 +99,20 @@ def collectors_from_sources_api
         hash
       end
 
-      def internal_url_for(path, query = nil)
-        URI.parse(api_base_url).tap do |uri|
-          uri.path = File.join("/internal/v0.0/", path)
-          uri.query = query if query
-        end.to_s
+      def sources_api_url_for(path)
+        File.join(sources_url, ENV["PATH_PREFIX"].to_s, ENV["APP_NAME"].to_s, SOURCES_API_VERSION, path)
```

Comment:
The `APP_NAME` value shouldn't be the same for sources and topo. I think we can probably hardcode these here like we do for the version. They don't seem likely to change at this point.

Reply:
Hm you're right, let me just move building the api path out of this and provide a full url to the sources and topology api, since hard-coding these to "sources" and "topological-inventory" would break the ability to run locally.